src/interfaces/current-account-interface.ts

```diff
@@ -1,8 +1,12 @@
-// Account information interface
+/** The ROOT state model interface. */
```

Comment:
I'm not sure what you mean by "root" state model. This is the interface for the current account Vuex module, right?

Reply:
Hmm, actually, no, it's in fact the currentAccount object in the account module... 🤔